### ISSUE TITLE
fluent-debug doesn't work in v0.12.26 even with #894

### ISSUE THREAD

**Issue by gunyarakun** [fluent/fluentd#1035]
## Environments

- version: fluentd version 0.12.26
- OS: Alpine Linux

## My configuration

```
<source>
  @type forward
  port 24224
</source>

<source>
  @type debug_agent
</source>

<match **>
  @type stdout
</match>
```

## My problem explanation

#894 is a backport of fixing `fluent-debug` to work for v0.14.
Event though after the commit, I cannot execute `fluent-debug` successfully.

For example, in `lib/fluent/command/debug.rb`, the line

```ruby
include Fluent::SystemConfig::Mixin
```
on https://github.com/fluent/fluentd/blob/v0.12.26/lib/fluent/command/debug.rb#L67
should be 
```ruby
include Fluent::SystemConfigMixin
```
as far as I see the line https://github.com/fluent/fluentd/blob/v0.12.26/lib/fluent/system_config.rb#L21

Or, I think `lib/fluent/system_config.rb` should be modified to define `Fluent::SystemConfig::Mixin` like v0.14.

**Comment by repeatedly** [comment]
Thanks for the report.
I will fix it later.

**Comment by repeatedly** [comment]
Fixed at v0.12 branch!
